### PR TITLE
Pileup summary info slimmer for MiniAOD (76X)

### PR DESCRIPTION
**Pull request by lgray**
Forward port of #10592.

### CODE REVIEW

**Comment by lgray** [comment]
@cmsbuild please test

**Comment by lgray** [comment]
@monttj can you give a +1 here too? Thanks!

**Comment by monttj** [comment]
+1

**Comment by davidlange6** [comment]
+1

File: PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py [hunk 1]
```diff
@@ -3,6 +3,9 @@
 from PhysicsTools.SelectorUtils.tools.vid_id_tools import *
 
 def miniAOD_customizeCommon(process):
+    #slimmed pileup information
+    process.load('PhysicsTools.PatAlgos.slimming.slimmedAddPileupInfo_cfi')
```
Comment (slava77) [inline PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py]: the way this module is currently implemented, it should be only in miniAOD_customizeMC
PileupSummaryInfo itself would not fit the info available in real data

**Pull request merged**